rsocket/internal/SetupResumeAcceptor.cpp

```diff
@@ -50,6 +54,9 @@ class OneFrameProcessor : public FrameProcessor {
   void onTerminal(folly::exception_wrapper ew) override {
     onSetup_ = nullptr;
     onResume_ = nullptr;
+    if(!transport_) {
```

Comment:
We may move this check into acceptor_.close(..) and let the decision to be given by that object.